packages/ag-charts-community/src/options/chart/errorBarOptions.ts

```diff
@@ -1,10 +1,47 @@
-export interface AgErrorBarOptions {
+import type { StrokeOptions } from '../series/cartesian/commonOptions';
+import type { PixelSize, Ratio } from './types';
+
+interface ErrorBarStylingOptions extends StrokeOptions {
+    /** Whether to display the error bars. */
+    visible?: boolean;
+}
+
+interface ErrorBarCapLengthOptions {
+    /** Absolute length of caps in pixels. */
+    length?: PixelSize;
+    /** Length of caps relative to the shape used by the series. */
+    lengthRatio?: Ratio;
+}
+
+export interface AgErrorBarDataOptions {
+    /** The key to use to retrieve lower bound error values from the x axis data. */
+    xLowerKey?: string;
+    /** Human-readable description of the lower bound error value for the x axis. This is the value to use in tooltips or labels. */
+    xLowerName?: string;
+    /** The key to use to retrieve upper bound error values from the x axis data. */
+    xUpperKey?: string;
+    /** Human-readable description of the upper bound error value for the x axis. This is the value to use in tooltips or labels. */
+    xUpperName?: string;
+
     /** The key to use to retrieve lower bound error values from the y axis data. */
-    yLowerKey: string;
+    yLowerKey?: string;
     /** Human-readable description of the lower bound error value for the y axis. This is the value to use in tooltips or labels. */
     yLowerName?: string;
     /** The key to use to retrieve upper bound error values from the y axis data. */
-    yUpperKey: string;
+    yUpperKey?: string;
     /** Human-readable description of the upper bound error value for the y axis. This is the value to use in tooltips or labels. */
     yUpperName?: string;
 }
+
+export interface AgErrorBarCapOptions extends ErrorBarCapLengthOptions, ErrorBarStylingOptions {}
+
+export interface AgErrorBarOptions extends AgErrorBarDataOptions, ErrorBarStylingOptions {
+    /** Options to style error bars' caps */
+    cap?: AgErrorBarCapOptions;
+}
+
+export interface AgErrorBarSeriesOptions {
+    // TODO: relative URL ./error-bars resolves to /error-bars instead of /[lang]/error-bars
```

Comment:
We should consider addressing this bug. I feel that including links to the error bars page within the line, bar, and scatter series pages would enhance the documentation's overall navigation

Reply:
So I wouldn't include links like this in the JSDocs - these JSDocs are presented in IDE auto-complete for users, and the Markdown style links don't make any sense there?

Reply:
That's unfortunate. In that case it would be better to remove these Markdown links for now.

But we may want to revisit this feature at some point, I've logged a backlog issue for this: https://ag-grid.atlassian.net/browse/AG-9562

Reply:
Link removed.